cmd/juno/juno.go

```diff
@@ -108,8 +108,8 @@ const (
 	defaultPprof                              = false
 	defaultPprofPort                          = 6062
 	defaultColour                             = true
-	defaultPendingPollInterval                = 5 * time.Second
-	defaultPreConfirmedPollInterval           = time.Second
+	defaultPendingPollInterval                = time.Second
+	defaultPreConfirmedPollInterval           = 500 * time.Millisecond
```

Comment:
I believe we might have to update our docs here as well:
juno.nethermind.io/configuring#configuration-options